types/node/index.d.ts

```diff
@@ -555,6 +555,12 @@ declare namespace NodeJS {
         stderr: WriteStream;
         stdin: ReadStream;
         openStdin(): Socket;
+        /**
+         * @deprecated
+         * Using process.binding() in general should be avoided.
+         * The type checking methods in particular can be replaced by using util.types.
```

Comment:
It's not clear to me what this comment should tell me. What related to `process.binding` should be replaced by `util.types`?